src/controllers/users.js

```diff
@@ -1,4 +1,6 @@
 import User from '@models/User';
+const jwt = require('jsonwebtoken');
+const accessToken = require('crypto').randomBytes(64).toString('hex')
```

Comment:
Generating a random key to sign the JWT might not be a good idea.

It would be better to get the signing key from `.env` variables